content/SCALE/SCALEUIReference/Network/NetworkInterfaceScreens.md

```diff
@@ -27,13 +40,13 @@ Use **Apply** to save your setting changes.
 ### Interface Settings
 These settings display for all interface types. The **Type** setting is only available and required on the **Add Interface** configuration screen.
 
-![AddInterfaceInterfaceSettings](/images/SCALE/22.02/AddInterfaceInterfaceSettings.png "Interface Settings")
+![AddInterfaceInterfaceSettings](/images/SCALE/23.10/AddInterfaceInterfaceSettings.png "Interface Settings")
 
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
 | **Type** | Required field. Select the type of interface from the dropdown list or options **Bridge**, **Link Aggregation** or **VLAN**. Each option displays additional configuration settings for that type.<br> Select **Bridge** to create a logical link between multiple networks.<br> Select **Link Aggregation** to combine multiple network connections into a single interface.<br> Select **Virtual LAN** (VLAN) to partition and isolate a segment of the connection.<br> This field does not display on the **Edit Interface** screen. |
```

Comment:
We have not standardized on using just "Required". or "Required field." I'm partial to using just "required". thoughts?
```suggestion
| **Type** | Required. Select the type of interface from the dropdown list or options **Bridge**, **Link Aggregation** or **VLAN**. Each option displays additional configuration settings for that type.<br> Select **Bridge** to create a logical link between multiple networks.<br> Select **Link Aggregation** to combine multiple network connections into a single interface.<br> Select **Virtual LAN** (VLAN) to partition and isolate a segment of the connection.<br> This field does not display on the **Edit Interface** screen. |
```